### PR TITLE
Add tox test to check for invalid playbook include

### PR DESCRIPTION
**Pull request by mtnbikenc**
Playbooks should not include component entry point playbooks. This
results in running initialization steps again causing unintended
consequences.

Also brings back the previous check for entry points in private or common directories which could indicate orphaned files.

Removed orphaned and unused validator.yml files.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -340,11 +338,45 @@ def run(self):
...
         if not has_errors:
             print('...PASSED')
+
+        all_playbooks, included_playbooks = find_playbooks()
+
+        print('#' * 60)
+        print('Invalid Playbook Include Checks')
+        invalid_include = []
+        for playbook in sorted(included_playbooks):
+            ignored = ['common', 'private', 'init', 'aws', 'gcp', 'openstack',
```
Comment (michaelgugino) [inline setup.py]: We have entry points in common/openshift-cluster/upgrades/...

I'm not sure how this logic is working.  Could you add a comment?

File: setup.py [hunk 2]
```diff
@@ -340,11 +338,45 @@ def run(self):
...
+                       'deploy_cluster.yml', 'prerequisites.yml']
+            if any(x in playbook for x in ignored):
+                continue
+            invalid_include.append(playbook)
+        if len(invalid_include) > 0:
```
Comment (michaelgugino) [inline setup.py]: Dont' need to check len, can just do
```python
if invalid_include:
```

File: setup.py [hunk 3]
```diff
@@ -340,11 +338,45 @@ def run(self):
 
         if not has_errors:
             print('...PASSED')
+
+        all_playbooks, included_playbooks = find_playbooks()
+
+        print('#' * 60)
+        print('Invalid Playbook Include Checks')
+        invalid_include = []
+        for playbook in sorted(included_playbooks):
```
Comment (michaelgugino) [inline setup.py]: No need to sort a set.

File: setup.py (hunk 1 above)
Comment (michaelgugino) [inline setup.py]: Immutable list could be a tuple.

Comment (mtnbikenc) [reply setup.py]: We should not have entry points in common/openshift-cluster/upgrades.  The entry points for those playbooks are in byo/openshift-cluster/upgrades.  I've added a verbose comment to the code to describe why each item is being ignored.

File: setup.py (hunk 2 above)
Comment (mtnbikenc) [reply setup.py]: Updated

File: setup.py (hunk 3 above)
Comment (mtnbikenc) [reply setup.py]: Was sorting for ordered output.  Move the sort to the resultant list if it exists.

File: setup.py (hunk 1 above)
Comment (mtnbikenc) [reply setup.py]: I've updated all of them.

**Pull request merged**